SPECS/python-apache-libcloud/python-apache-libcloud.spec

```diff
@@ -0,0 +1,1070 @@
+#
+# spec file for package python-apache-libcloud
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+%global _description \
+Apache Libcloud is a standard Python library that abstracts away \
+differences among multiple cloud provider APIs.
+%global pypi_name apache-libcloud
+
+Name:           python-apache-libcloud
+Version:        3.5.1
+Release:        2
+Summary:        Abstraction over multiple cloud provider APIs
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://libcloud.apache.org
+Source0:        https://github.com/apache/libcloud/archive/refs/tags/v3.5.1.tar.gz#/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+Patch1:         gce_image_projects.patch
+Patch2:         ec2_create_node.patch
+Patch3:         skip-some-tests-for-older-paramiko-versions.patch
+# PATCH-FIX-UPSTREAM https://github.com/Kami/libcloud/commit/e62bb28cdbd685203d44a9a4028f311ea155476c Use unittest.mock library from stdlib instead of using 3rd party mock dependency.
+Patch4:         mock.patch
+
+BuildRequires:  fdupes
+BuildRequires:  libvirt-python3
+BuildRequires:  python3-lockfile
+BuildRequires:  python3-lxml
+BuildRequires:  python3-paramiko
+BuildRequires:  python3-pyOpenSSL
+BuildRequires:  python3-requests-mock
+BuildRequires:  python3-setuptools
+BuildRequires:  pyproject-rpm-macros
+Requires:       python3-lxml
+Requires:       python3-requests
+
+%if %{with_check}
+BuildRequires:  python3-pip
+BuildRequires:  python3-pytest
+%endif
+
+%description %{_description}
+
+%package -n     python3-%{pypi_name}
+Summary:        %{summary}
+
+%description -n python3-%{pypi_name} %{_description}
+
+%prep
+%autosetup -p1 -n %{pypi_name}-%{version}
+sed -i '/^#!/d' demos/gce_demo.py
+chmod a-x demos/gce_demo.py
+# Setup tests
+cp libcloud/test/secrets.py-dist libcloud/test/secrets.py
+
+%build
+%py3_build
+
+%install
+%py3_install
+find %{buildroot} -name '*.DS_Store' -delete
+find %{buildroot} -name '*.json' -size 0 -delete
+find %{buildroot} -name '*.pem' -size 0 -delete
+rm -r %{buildroot}%{python3_sitelib}/libcloud/test
+%fdupes %{buildroot}%{python3_sitelib}
+
+%check
+%{python3} -m pip install atomicwrites attrs pluggy pygments six more-itertools
+# Skip OvhTests::test_list_nodes_invalid_region which tries to reach OVH servers
```

Comment:
We have internet access during test builds, so skipping this test shouldn't be necessary

Reply:
Not skipping anymore.